west.yml

```diff
@@ -134,6 +133,10 @@ manifest:
       repo-path: sdk-cjson
       path: modules/lib/cjson
       revision: 50bddddf7fb45e4ce87bc3619df06db55d181856
+    - name: openthread
+      repo-path: sdk-openthread
+      path: modules/lib/openthread
+      revision: 6e2ab99d68f5cd4c6ad5d728b7c7baadfcd22429
```

Comment:
This looks like the wrong commit. Tip is actually pointing at 3908fb878493367a23503548049d47a138193144, which is a merge commit bringing in this commit. @rlubos was the merge commit created intentionally?

Reply:
> @rlubos was the merge commit created intentionally?

I am going to assume "yes", keep this as-is, and force push tip backwards to point to this one

Reply:
Actually it was, I've assumed the same process we have in `zephyrproject-rtos/openthread` for bringing upmerges (i.e. to do this with GH UI).